include/irq.h

```diff
@@ -127,6 +129,10 @@ int32_t irq_global_enable(struct irq_ctrl_desc *desc);
 /* Global interrupt disable */
 int32_t irq_global_disable(struct irq_ctrl_desc *desc);
 
+/* Set interrupt trigger level. */
+int32_t irq_trigger_level_set(struct irq_ctrl_desc *desc, uint32_t irq_id,
```

Comment:
Can we set it from init param?

Reply:
We can. Should we have this too? In my opinion yes, but I don't know.

Reply:
Did not add it yet as that would change the way the driver needs to be initialized. Are we sure we want to change that in this PR?